### PR TITLE
All variables in an alias set must have the same Dimensions

### LINKED ISSUE #731: Aliases and Dimensions for array variables
Presumably all variables of in the same alias set (ie. with the same valueReference) would have to have the same Dimensions settings, if the value is an array? If so, this should be added to section 2.2.10 where the restrictions on aliases are described.

### PR DESCRIPTION
**Pull request by nickbattle**
Resolves #731

### CODE REVIEW

**Comment by chrbertsch** [comment]
FMI regular design web meeting:

Torsten S: Can we simplify the wording? Is "alias set" defined?
Andreas P: I think "alias set" is used even in FMI 2.0
Torsten S: "Alias set" appears two times in FMI 3.0
Andreas P.: But it is not defined.
Torsten S: one should write: "all variables sharing the same value reference"
Torsten S: Lets define "alias set".
Klaus: currently the definition of "alias" is too sloppy ...  @klausschuch : I will create a ticket for this.
Andreas: Could you then also fix #729 ?
Klaus: Perhaps we should change the definition of aliases: one variable has the information of dimensions, the other alias variables only have other name. Otherwise one has to check all the attributes of the dimensions of the different variables ...
Andreas P: We have a similar topic with start values of const variables, on  p.52 of FMI 2.0
Christian: Should an alias variable then have an attribute "alias"?

**Comment by t-sommer** [comment]
superseded by #748
